src/index.js

```diff
@@ -40,24 +40,48 @@ class LocalStorage extends Service {
   
   execute(method, ... args) {
     return this.ready()
-      .then(() => super[method](... args))
-      .then(data => this.flush(data));
+      .then(() => super[method](... args));
+  }
+
+  get(... args) {
+    return this.execute('get', ... args);
+  }
+
+  find(... args) {
+    return this.execute('find', ... args);
+  }
+
+  // Create without hooks and mixins that can be used internally
+  _create(data) {
+    let id = data[this._id] || (this._uId + 1);
+
+    // If the item already exists then just update it.
+    if (this.store[id]) {
+      return this.update(id, data);
```

Comment:
Ya possibly. After I thought about it I think we can treat feathers-memory more like a database.